test/bigtest/tests/QuickMarcEditor-test.js

```diff
@@ -63,4 +63,25 @@ describe('Quick MARC editor', () => {
       expect(quickMarcEditor.editorRows().length).to.be.equal(records.length + 2);
     });
   });
+
+  describe('after pressing remove button', () => {
+    beforeEach(async function () {
+      await quickMarcEditor.removeButton.click();
+    });
+
+    it('should be render confirmation modal', () => {
+      expect(quickMarcEditor.removeConfirmation.isPresent).to.be.true;
+    });
+  });
+
+  describe('remove record', () => {
```

Comment:
what about this?

Reply:
@NikitaSedyx  what is wrong in this case? We just  remove record in this test

Reply:
name, you updated previous one, why not to make case more readable?